class GetException(Exception):
    pass

class RulesException(Exception):
    pass